documents/Project_idea_document.md

```diff
@@ -45,6 +45,47 @@ The possibility to sell the ideas to companies was mentioned. I prefer the idea
 
 ####Requirements and Specifications
+
+#####a)	User Characteristics
+The main users that the application is created for are both experienced and non-experienced users; it can be used by average users that have ideas that they would like to be made but do not have the required time/skills/money for it. The general users that the application addresses are listed below:
+* 	Non-experienced User:	Average user with little or no expertise. Uses the system to share ideas and/or vote on ideas.
+* Experienced User:	User with expertise in the field, who uses the system to take the ideas to the implementation level. They can be in-house experts who define the chosen ideas. They can also vote on ideas.
+* Investors/Companies:	Users who are looking for new ideas to get implemented and gain profit by funding and/or providing resources for it. They can also vote on ideas that they would like to get implemented.	
+
+#####b)	Functional Requirements
+The application functions in two phases; one for sharing and voting ideas and another for working on the implementation of the ideas. The main functional requirements in these phases are as follows:
+
+Phase 1:
+*	The system allows users to sign up and create a user profile.
+*	The system must allow users to submit ideas for mobile applications, accompanied by a short text describing the idea and an optional picture.
+*	The system must put the idea along with its description through a sanitation check to eliminate submissions that are inappropriate according to specified rules. Sanitation check is performed by CrowdFlower.
+*	If a picture is also provided with the submitted idea, the system puts the picture through the sanitation check as well.
+*	The system must give a notification informing the user of the status of their submission; it can either be accepted or rejected. If it is rejected, a reason must be provided to the user.
+*	If the submission is accepted, the system transfers the submitted idea to the public pool.
+*	The system must allow all users to see the ideas in the public pool and vote on them.
+*	If an idea gets enough votes, the system takes it to Phase 2.
+
+Phase 2:
+*	Ideas are clearly defined by in-house experts.
+*	Appropriate investors and companies are informed of the idea to implement it.
+*	Users can choose to get their ideas developed in-house or find outside developers.
+*	Users who originally submit the ideas get a certain percentage from the profit.
```

Comment:
@HDking do you think we should define the percentage of the profit the original user gets for the idea?